core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -739,6 +741,8 @@ void populate1() {
           NullPolicy.STRICT);
       defineMethod(BITXOR, BuiltInMethod.BIT_XOR.method,
           NullPolicy.STRICT);
+      defineMethod(LEFTSHIFT, LEFT_SHIFT.method,
+          NullPolicy.STRICT);
```

Comment:
Right between two methods that start with BIT??! really? 

Use the BuilInMethod qualifier, like the other code.